src/components/views/settings/tabs/user/AppearanceUserSettingsTab.tsx

```diff
@@ -250,15 +252,19 @@ export default class AppearanceUserSettingsTab extends React.Component<IProps, I
             <div className="mx_SettingsTab_section mx_AppearanceUserSettingsTab_themeSection">
                 <span className="mx_SettingsTab_subheading">{_t("Theme")}</span>
                 {systemThemeSection}
-                <Field
-                    id="theme" label={_t("Theme")} element="select"
-                    value={this.state.theme} onChange={this.onThemeChange}
+                <div className="mx_ThemeSelectors" onChange={this.onThemeChange}>
+                    {orderedThemes.map(theme => {
+                        return <StyledRadioButton
```

Comment:
needs a `key`

Feels a bit strange to have an infinite-unbounded list as not a `select` as it was

Reply:
Thanks.

Yes I know, it could be a select but then we'd have to implement the styling for the radio buttons twice. And though technically infinite the number of themes loaded is likely quite low

Reply:
Actually, the design shows the custom themes in Labs instead, so this should presumably map `builtInThemes` instead

Reply:
Custom themes haven't been moved to labs yet. Will change it to a map when I do that

Reply:
Actually, I'll do it now

Reply:
On third thought it makes more sense as a separate pr